website/pages/en/subgraphs/querying/introduction.mdx

```diff
@@ -2,20 +2,30 @@
 title: Querying The Graph
 ---
 
-When a subgraph is published to The Graph Network, you can visit its subgraph details page on [Graph Explorer](https://thegraph.com/explorer) and use the "query" tab to explore the deployed GraphQL API for the subgraph, issuing queries and viewing the schema.
+To start querying right away, visit [The Graph Explorer](https://thegraph.com/explorer).
 
-> Please see the [Query API](/subgraphs/querying/graphql-api/) for a complete reference on how to query the subgraph's entities. You can learn about GraphQL querying best practices [here](/subgraphs/querying/best-practices/)
+## Overview
 
-> Note: If you encounter 405 errors with a GET request to the Graph Explorer URL, please switch to a POST request instead.
+When a subgraph is published to The Graph Network, you can visit its subgraph details page on [Graph Explorer](https://thegraph.com/explorer) and use the "query" tab to explore the deployed GraphQL API for each subgraph.
```

Comment:
Again, because we linked to it just above, I'm not sure this is necessary:
```suggestion
When a subgraph is published to The Graph Network, you can visit its subgraph details page on Graph Explorer and use the "query" tab to explore the deployed GraphQL API for each subgraph.
```